test.py

```
@@ -9094,3 +9094,1163 @@ async def test_cli_config_validate_fails_when_unconfigured(db_pool):
+        )
+        
+        # Try to insert duplicate - should be prevented by unique constraint
+        try:
+            await conn.execute(
+                """
+                INSERT INTO concepts (name, depth)
+                VALUES ($1, 0)
+                """,
+                concept_name,
+            )
+            assert False, "Duplicate concept name should be prevented"
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)